virttest/libvirt_vm.py

```diff
@@ -1341,7 +1366,13 @@ def add_cputune(vcpu_cputune=""):
             virt_install_cmd += " --extra-args '%s'" % kernel_params
 
         virt_install_cmd += " --noautoconsole"
-
+        # Add TPM device
+        tpm_device = params.get("tpm_device_path", None)
+        if tpm_device:
+            tpm_model = params.get("tpm_model", None)
+            tpm_type = params.get("tpm_type", None)
+            virt_install_cmd += add_tpmdevice(help_text, tpm_device, tpm_model,
```

Comment:
And if the function does return 'None', this line would fail, since 'virt_install_cmd' is a str, and cannot be '+' by 'None'.

Reply:
makes sense, tnx, will change it.